pkg/deploy/assets/rp-production.json

```diff
@@ -1017,6 +1017,67 @@
                 "[resourceId('Microsoft.DocumentDB/databaseAccounts', parameters('databaseAccountName'))]"
             ]
         },
+        {
+            "properties": {
+                "severity": 2,
+                "enabled": true,
+                "scopes": [
+                    "[resourceId('Microsoft.DocumentDB/databaseAccounts', parameters('databaseAccountName'))]"
+                ],
+                "evaluationFrequency": "PT5M",
+                "windowSize": "PT1H",
+                "targetResourceType": "Microsoft.DocumentDB/databaseAccounts",
+                "criteria": {
+                    "allOf": [
+                        {
+                            "operator": "GreaterThan",
+                            "threshold": 10,
+                            "AdditionalProperties": null,
+                            "name": "ThrottledRequestCheck",
+                            "metricName": "TotalRequests",
+                            "metricNamespace": "Microsoft.DocumentDB/databaseAccounts",
+                            "timeAggregation": "Count",
+                            "dimensions": [
+                                {
+                                    "name": "StatusCode",
+                                    "operator": "Include",
+                                    "values": [
+                                        "429"
+                                    ]
+                                }
+                            ],
+                            "criterionType": "StaticThresholdCriterion"
+                        },
+                        {
+                            "operator": "GreaterThan",
+                            "threshold": 90,
```

Comment:
Couple small follow ups as per PR review meeting (severity, confirm thresholds, confirm tsg, plan demo) but otherwise looks great and thank you for your work.